server/src/main/java/io/deephaven/server/console/ConsoleServiceGrpcImpl.java

```diff
@@ -255,25 +256,29 @@ public StreamObserver<AutoCompleteRequest> autoCompleteStream(
             if (AUTOCOMPLETE_DISABLED) {
                 return new NoopAutoCompleteObserver(session, responseObserver);
             }
-            if (PythonDeephavenSession.SCRIPT_TYPE.equals(scriptSessionProvider.get().scriptType())) {
-                PyObject[] settings = new PyObject[1];
-                safelyExecute(() -> {
-                    final ScriptSession scriptSession = scriptSessionProvider.get();
-                    scriptSession.evaluateScript(
-                            "from deephaven_internal.auto_completer import jedi_settings ; jedi_settings.set_scope(globals())");
-                    settings[0] = (PyObject) scriptSession.getVariable("jedi_settings");
-                });
-                boolean canJedi = settings[0] != null && settings[0].call("can_jedi").getBooleanValue();
-                log.info().append(canJedi ? "Using jedi for python autocomplete"
-                        : "No jedi dependency available in python environment; disabling autocomplete.").endl();
-                return canJedi ? new PythonAutoCompleteObserver(responseObserver, scriptSessionProvider, session)
-                        : new NoopAutoCompleteObserver(session, responseObserver);
+            final ScriptSession scriptSession = scriptSessionProvider.get();
+            if (PythonDeephavenSession.SCRIPT_TYPE.equals(scriptSession.scriptType())) {
+                final PyObject scope = ((PythonDeephavenSession) scriptSession).scope().mainGlobals().unwrap();
+                // noinspection EmptyTryBlock,unused
+                try (final JediCompleter _unused = jedi(scope)) {
+                    // ensure we can create it
+                } catch (RuntimeException e) {
+                    log.debug(e).append("Unable to create autocomplete; is jedi installed?").endl();
```

Comment:
will anybody ever see this?
if someone expects autocomplete but it doesn't work, I doubt they are going to start w/ debug logging, or even notice this amid the logspam..

Reply:
if we raise the log level so it's easily noticed, we can include a message about how to disable autocomplete altogether.